### PR TITLE
Cybereason Integration Enhancement (involved hashes)

### PR DESCRIPTION
**Pull request by Itay4**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/14931

## Description
Added affected machines and involved hashes to query malops outputs.

## Required version of Demisto
Any

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation - https://github.com/demisto/etc/issues/14946
- [x] Code Review

## Dependencies
None

### CODE REVIEW

File: Integrations/integration-Cybereason.yml [hunk 1]
```diff
@@ -518,6 +518,12 @@ script:
                 machine_name = machine.get('name', '')
                 affected_machines.append(machine_name)
 
+            involved_hashes = []
+            if 'rootCauseElementHashes' in malop['simpleValues']:
+                if malop['simpleValues']['rootCauseElementHashes']['totalValues'] != 0:
+                    for hashcake in malop['simpleValues']['rootCauseElementHashes']['values']:
```
Comment (yaakovi) [inline Integrations/integration-Cybereason.yml]: no need for the loop
```suggestion
                   involved_hashes.extend(malop['simpleValues']['rootCauseElementHashes']['values'])
```

File: Integrations/integration-Cybereason.yml [hunk 2]
```diff
@@ -526,7 +532,8 @@ script:
                 'Suspects':suspects_string,
                 'LastUpdateTime': malop_last_update_time,
                 'Status': management_status,
-                'AffectedMachines': affected_machines
+                'AffectedMachine': affected_machines,
```
Comment (yaakovi) [inline Integrations/integration-Cybereason.yml]: why the machines was changed to machine? (single)

File: Integrations/integration-Cybereason.yml [hunk 3]
```diff
@@ -1663,6 +1670,12 @@ script:
     - contextPath: Cybereason.Malops.LastUpdatedTime
       description: Last updated time of malop
       type: string
+    - contextPath: Cybereason.Malops.AffectedMachine
```
Comment (yaakovi) [inline Integrations/integration-Cybereason.yml]: Status is also missing in the context output

File: Integrations/integration-Cybereason.yml [hunk 4]
```diff
@@ -1663,6 +1670,12 @@ script:
...
+      description: List of machines affected by this Malop
+      type: string
+    - contextPath: Cybereason.Malops.InvolvedHash
+      description: List of file hashes involved in this Malop
+      type: string
```
Comment (yaakovi) [inline Integrations/integration-Cybereason.yml]: this is not a string, it's a list.

File: Integrations/integration-Cybereason.yml (hunk 2 above)
Comment (Itay4) [reply Integrations/integration-Cybereason.yml]: Our context standard is singular

File: Integrations/integration-Cybereason.yml (hunk 4 above)
Comment (Itay4) [reply Integrations/integration-Cybereason.yml]: It's a list of strings

File: Integrations/integration-Cybereason.yml [hunk 5]
```diff
@@ -518,6 +518,12 @@ script:
                 machine_name = machine.get('name', '')
                 affected_machines.append(machine_name)
 
+            involved_hashes = []
+            if 'rootCauseElementHashes' in malop['simpleValues']:
+                if malop['simpleValues']['rootCauseElementHashes']['totalValues'] != 0:
+                   involved_hashes.extend(malop['simpleValues']['rootCauseElementHashes']['values'])
+                        involved_hashes.append(hashcake)
```
Comment (yaakovi) [inline Integrations/integration-Cybereason.yml]: this needs to be removed

**Pull request merged**